### ISSUE TITLE
How to train only fc7 layer of face recognition model and keeping pretrained weights frozen?

### ISSUE THREAD

**Issue by hetul-patel** [deepinsight/insightface#1077]
I want to fine-tune res50 pretrained model on VGG2 dataset. The res50 is originally trained on MS1M. Fc7 layer's weights of res50 are not provided and only base network weights are available. I want to freeze pretrained weights and first only train classification layer for VGG2. Once classification loss is reduced then I want to train entire network. Can anyone help, how to update train.py script for freezing layers before fc7?

**Comment by orlevit** [comment]
Following

**Comment by wudrans** [comment]
@hetul-patel hi, do you have solved this problem? I also have the same problem, if you have solved, can you help me? thank you
